cmd/builder/app/server.go

```diff
@@ -32,6 +32,7 @@ func Run(ctx context.Context, logger *zap.Logger, mgr manager.Interface, shareVo
 	builder := builder.MakeBuilder(logger, shareVolume)
 	mux := http.NewServeMux()
 	mux.HandleFunc("/", builder.Handler)
+	mux.HandleFunc("/clean", builder.Clean)
```

Comment:
Please try once commenting on this line. 
Following scenario may happen,
It may happen user upgrades to newer version of fission but builder pod is old. 
In that case, clean API would not be present and client would get 404 response.
We should be able to handle this case, from client side.

Reply:
Even if clean API is missing, the request will be forwarded to `builder.Handler` since it accepts all request because of `/` route and we get a 500 response instead of 404.

Reply:
Even if i change the clean API method from `POST` to `DELETE`, still the response would 405 instead of 404.
Which approach should i take @sanketsudake ?
1. Handle 500 response in client.
2. Change clean API method from POST to DELETE.

I think approach 2 is good because in first approach builder will try to build the package but in 2nd approach it will return 405 response and avoid building the package.

Reply:
Also i think it would be better if we replace `/` route with `/build` route.

Reply:
> Change clean API method from POST to DELETE.
This one sounds better. We should change existing API behaviour for no reason.

Reply:
Done